src/pages/who-we-are/components/modal.tsx

```diff
@@ -7,7 +7,7 @@ import Linkedin from 'images/svg/who-we-are/linkedin.svg'
 
 type ModalPropsType = {
     name: string
-    position: string
+    position: string | ReactElement
```

Comment:
Having ReactElement as a type for position does not make sense.

Reply:
This type is because of the translations! we are using Localize component because of that I added this type for fixing ts errors
but I need to check that we need string type here or not